client/my-sites/people/people-notices/index.jsx

```diff
@@ -101,6 +101,21 @@ class PeopleNotices extends React.Component {
 					context: 'Error message after A site has failed to perform actions on a user.',
 				} );
 			case 'RECEIVE_DELETE_SITE_USER_FAILURE':
+				if ( 'user_owns_domain_subscription' === log.error.error ) {
+					return i18n.translate(
+						'@%(user)s owns following domain(s) used on this site: {{strong}}%(domains)s{{/strong}}. Domains owned by this user will have to be transferred to a different site, transferred to a different registrar, or canceled before removing or deleting @%(user)s.',
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?